changelog.md

```diff
@@ -1,5 +1,7 @@
 ## next [????.??.??]
+
 * Fixed cookie example from `Cookie` module documentation. `getCookie` Function would return strict variant of `Text`. Will convert it into lazy variant using `fromStrict`. 
+* Remove re-export of `Network.Wai.Parse.ParseRequestBodyOptions` from `Web.Scotty` and `Web.Scotty.Trans`. This is a type from `wai-extra` so exporting it is unnecessary.
```

Comment:
Could you list it under the Breaking Changes for "next"? Thanks!

Reply:
Moved it.